flixel/FlxG.hx

```diff
@@ -500,6 +505,43 @@ class FlxG
 		width = Std.int(Math.abs(Width));
 		height = Std.int(Math.abs(Height));
 		
+		renderMethod = BLIT;
+		
+		#if (!lime_legacy && !flash)
+			if (Lib.application.config.windows[0].hardware == false)
+			{
+				renderMethod = BLIT;
+			}
+			else
+			{
+				renderMethod = switch(stage.window.renderer.type)
+				{
+					case OPENGL, CONSOLE:      TILES;
+					case CANVAS, FLASH, CAIRO: BLIT;
+					default:                   BLIT;
+				}
+			}
+		#else
+			#if (flash || js)
+				renderMethod = BLIT;
+			#else
+				renderMethod = TILES;
+			#end
+		#end
+		
+		#if FLX_RENDER_BLIT
+			//renderMethod = BLIT;
```

Comment:
Previously the compiler constants would be set by looking at your targets: BLIT for Flash or JS, TILES for cpp, and all the logic seemed to be assuming you were using legacy.

This gives it a better way to guess at initial settings. In next mode, OpenGL and Console rendering should use tiles by default, and HTML5 with canvas, Flash, or CPP with cairo, should use blitting. In legacy mode, it reproduces the same logic as before.

Or maybe you're just asking about the last line? This block?
```
#if FLX_RENDER_BLIT
	//renderMethod = BLIT;
#elseif FLX_RENDER_TILE
	//renderMethod = TILES;
#end
```

In that case that's some old cruft, the first thing I did was just set runtime flags triggered by the compile constant as an early step in the refactor... let me delete that.